arlo_server/__init__.py

```diff
@@ -398,7 +398,8 @@ def audit_basic_update(election_id):
     election.online = info['online']
 
     errors = []
-    db.session.query(TargetedContest).filter_by(election_id=election.id).delete()
+    if info['contests']:
+        db.session.query(TargetedContest).filter_by(election_id=election.id).delete()
```

Comment:
There should be a backend test that verifies that this endpoint can be called without `contests` and that it just updates the `Election`.

Reply:
@eventualbuddha Agreed, definitely. Should that be added into this PR by someone with more backend experience?